pootle/apps/pootle_fs/management/commands/init_fs_project.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import logging
+
+from django.core.management import BaseCommand, CommandError
+from django.db import IntegrityError
+
+from pootle_fs.utils import FSPlugin
+from pootle_language.models import Language
+from pootle_project.models import Project
+
+
+logger = logging.getLogger('pootle.fs')
+
+
+class Command(BaseCommand):
+    help = "Init a new Pootle FS project."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'code',
+            metavar='CODE',
+            help='Project code'
+        )
+        parser.add_argument(
+            'fs',
+            metavar='FS_URL',
+            help='FS url "protocol_code+/repo/path/"'
+        )
+        parser.add_argument(
+            'translation_path',
+            help='Translation path "<language_code>/<filename>.<ext>"',
+            metavar='TRANSLATION_PATH'
+        )
+        parser.add_argument(
+            '--name',
+            action='store',
+            dest='name',
+            nargs='?',
```

Comment:
If I understand correctly doesn't this mean that ``--name`` and ``--name="My Project"`` are both valid?